src/commands/sbom/language.ts

```diff
@@ -0,0 +1,23 @@
+import {DependencyLanguage} from './types'
+
+// Attempt to find the language from a SBOM component. For now, we get the source either from
+// the bom-ref or the purl property of the SBOM.
+export const getLanguageFromComponent = (component: any): DependencyLanguage | undefined => {
+  const componentName = component['name']
+
+  if (component['bom-ref']) {
+    if (component['bom-ref'].includes('pkg:npm')) {
+      return DependencyLanguage.NPM
+    }
+    if (component['purl'].indexOf('pkg:composer') !== -1) {
+      return DependencyLanguage.PHP
+    }
+    if (component['purl'].indexOf('pkg:cargo') !== -1) {
+      return DependencyLanguage.RUST
+    }
```

Comment:
Sorry for not saying it, but I also meant to change those other `indexOf()`
```suggestion
    if (component['purl'].includes('pkg:composer')) {
      return DependencyLanguage.PHP
    }
    if (component['purl'].includes('pkg:cargo')) {
      return DependencyLanguage.RUST
    }
```

Reply:
done!